modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/dht/GridDhtTxLocal.java

```diff
@@ -94,7 +94,7 @@ public class GridDhtTxLocal extends GridDhtTxLocalAdapter implements GridCacheMa
     private volatile GridDhtTxPrepareFuture prepFut;
 
     /** Transaction label. */
-    private @Nullable String lb;
+    private final @Nullable String lb;
```

Comment:
Looks like `lb`, `prepFut` and `PREP_FUT_UPD` can be moved to upper class.

Reply:
Sure, but current goal is just to apply IDEA's tips.

Reply:
The goal looks strange. It's a small refactoring anyway, why both of these refactorings can't be done together?